pandas/io/parquet.py

```diff
@@ -252,7 +258,8 @@ def to_parquet(df, path, engine='auto', compression='snappy', index=None,
     ----------
     df : DataFrame
     path : string
-        File path
+        File path ( Will be used as Directory path if
```

Comment:
Can remove the parenthesis, replace with a new sentence.

Rather than "Directory path", say "`root_path`", which is the name used by `pyarrow.parquet.write_to_dataset`.